app/2.2.x/upgrading.md

```diff
@@ -0,0 +1,3274 @@
+---
+title: Upgrade guide
+skip_read_time: true
+---
+
+This document guides you through the process of upgrading Kong. First, check if
+a section named "Upgrade to `x.x.x`" exists, with `x.x.x` being the version
+you are planning to upgrade to. If such a section does not exist, the upgrade
+you want to perform does not have any particular instructions, and you can
+simply consult the [Suggested upgrade path](#suggested-upgrade-path).
+
+## Suggested upgrade path
+
+Unless indicated otherwise in one of the upgrade paths of this document, it is
+possible to upgrade Kong **without downtime**:
+
+Assuming that Kong is already running on your system, acquire the latest
+version from any of the available [installation
+methods](https://getkong.org/install/) and proceed to install it, overriding
+your previous installation.
+
+If you are planning to make modifications to your configuration, this is a
+good time to do so.
+
+Then, run migration to upgrade your database schema:
+
+```shell
+$ kong migrations up [-c configuration_file]
+```
+
+If the command is successful, and no migration ran
+(no output), then you only have to
+[reload](https://docs.konghq.com/1.0.x/cli/#kong-reload) Kong:
+
+```shell
+$ kong reload [-c configuration_file]
+```
+
+**Reminder**: `kong reload` leverages the Nginx `reload` signal that seamlessly
+starts new workers, which take over from old workers before those old workers
+are terminated. In this way, Kong will serve new requests via the new
+configuration, without dropping existing in-flight connections.
+
+
+## Upgrade to `2.1.0`
```

Comment:
@gszr where is upgrade to 2.2?

Reply:
I've added instructions for the 2.2 upgrade, thanks

Reply:
added here https://github.com/Kong/docs.konghq.com/pull/2413/commits/c0d2d002c9cc83aec5b3cf3108b843a9aff2206e